google/cloud/apigeeregistry/v1/admin_service.proto

```diff
@@ -105,6 +117,23 @@ message Status {
   string message = 1;
 }
 
+// Request message for MigrateDatabase.
+message MigrateDatabaseRequest {
+  // A string describing the kind of migration to perform.
+  // Currently only "auto" is recognized (and is the default if omitted).
+  string kind = 1;
+}
+
+// Metadata message for MigrateDatabase.
+message MigrateDatabaseMetadata {
+}
+
+// Response message for MigrateDatabase.
+message MigrateDatabaseResponse {
+  // A string describing the result of the migration.
+  string message = 1;
```

Comment:
What do you expect this field to indicate in the future? Right now I see it's just set to "OK" but that doesn't seem to provide any more information than the response status code.

Reply:
I think a useful improvement would be to report whether anything changed in the migration.